File: Try_GameEngine5/src/com/example/try_gameengine/utils/ISpriteDetectAreaHandler.java
package com.example.try_gameengine.utils;

/**
 * {@code ISpriteDetectAreaHandler} is use for deal with detect area controller.
 * @author irons
 *
 */
public interface ISpriteDetectAreaHandler {
	public void setSpriteDetectAreaBehavior(ISpriteDetectAreaBehavior spriteDetectAreaBehavior);
	public ISpriteDetectAreaBehavior getSpriteDetectAreaBehavior();
}

File: Try_GameEngine5/src/com/example/try_gameengine/utils/ISpriteDetectAreaHandler.kt
package com.example.try_gameengine.utils

/**
 * `ISpriteDetectAreaHandler` is use for deal with detect area controller.
 * @author irons
 // */
interface ISpriteDetectAreaHandler {
    fun getSpriteDetectAreaBehavior(): ISpriteDetectAreaBehavior?
    fun setSpriteDetectAreaBehavior(spriteDetectAreaBehavior: ISpriteDetectAreaBehavior?)
}

